src/helpers/propHelpers.ts

```diff
@@ -0,0 +1,31 @@
+import { PageSettings } from "../calendly";
+
+function getFirstCharacter(str: string): string {
+  const strArray: string[] = str.split("");
+  return strArray[0];
+}
+
+function sanitizeColorString(str: string): string {
+  if (getFirstCharacter(str) === "#") {
+    str = str.slice(1);
+  }
+  return str;
+}
+
+export function sanitizePageSettingsProps(
```

Comment:
Thanks for the quick PR, the code works great! Only requested change would be to call this function in the [`formatCalendlyUrl`](https://github.com/tcampb/react-calendly/blob/master/src/calendly.tsx#L105) method instead of the individual components.

Reply:
updated! Let me know how that looks, good call out